src/abilities/Uncle Fungus.js

```diff
@@ -108,8 +108,16 @@ G.abilities[3] = [
 		// activate() :
 		activate: function(target, args) {
 			var ability = this;
+			var uncle = this.creature;
 			ability.end();
-
+			
+			// If max health or more
+			if(uncle.stats.health >= uncle.baseStats.health)
+			{
+				//burp
+				uncle.hint("burp!", "msg_effects");
```

Comment:
Capitalized.

Reply:
please dont use K&R style braces. They should always follow the operator that they are tied to `if (....) {` `do {` `while (....) {` etc.

Reply:
@ktiedt Good find, I missed that one. Was rushing to get some sleep xD